pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -1,130 +1,168 @@
+### Setting up USWDS in your Sass entry point
+If your project does not yet have a Sass entry point, create a file called `index.scss` or `styles.scss`.
 
-Your project’s Sass entry point is a simple file that only needs to do the following three tasks:
-1. Include a USWDS [settings file]({{ site.baseurl }}/documentation/settings/)
-2. Include the USWDS source code
-3. Include your project’s custom Sass
+ Your project’s Sass entry point is a simple file that will only need to do the following three tasks:
 
-Create an entry point called something like `index.scss` or `styles.scss`. Your entry point should read like the following example:
+1. Load or define [USWDS settings]({{ site.baseurl }}/documentation/settings/) (required)
+2. Load USWDS source code (required)
+3. Load your project’s custom Sass (optional)
+
+For your entry point to complete these tasks, you will need to add the following references to your entry point in this order:
 
-{:.site-terminal}
 ```scss
-// Include a USWDS settings file (required)
+/* styles.scss */
+
+// 1. Load USWDS settings
 @forward "uswds-settings.scss";
 
-// Point to the USWDS source code (required)
+// 2. Load USWDS source code
 @forward "./path/to/source/uswds";
 
-// Include your project's custom Sass (optional)
+// 3. Load your project's custom Sass
 @forward "project-custom.scss";
 ```
 
+Note that each `@forward` reference in this example is tied to one of the tasks listed above.
+
 In plain language, this code says:
 
-- **Get the instructions**: Get the USWDS settings that tell the design system how to build the styles. Settings are the first thing you need to include.
+1. **Get the instructions**: Get the USWDS settings that tell the Design System how to build the styles. Settings are the first thing you'll need to include in your entry point.
 
-  Any individual setting is a Sass variable. Each setting begins with the `$theme-` prefix, and we provide a list of all available settings [in the USWDS documentation]({{ site.baseurl }}/documentation/settings/). The following is an example of a simple settings file:
+    In USWDS, settings are Sass variables that can be configured via Sass' `@use...with()` rule. You can find full instructions for configuring USWDS settings, along with a list of all available settings, on the [Settings page]({{ site.baseurl }}/documentation/settings/).
 
-  {:.site-terminal}
-  ```scss
-  @use "uswds-core" with (
-    $theme-image-path: "../uswds/img",
-    $theme-show-compile-warnings: true,
-    $theme-banner-background-color: "ink",
-    $theme-banner-link-color: "primary-light",
-    $theme-banner-max-width: "none",
-  )
-  ```
+2. **Create the foundation**: Build all USWDS styles from these settings.
 
-- **Create the foundation**: Build all USWDS styles from these settings.
+    Once you have set up your project's custom settings, it is time to pull in the the USWDS source code from the `uswds` package.
```

Comment:
Extra `the` towards end of sentence.

```suggestion
    Once you have set up your project's custom settings, it is time to pull in the USWDS source code from the `uswds` package.
```

Reply:
Good catch. Updated in [da0713f](https://github.com/uswds/uswds-site/pull/1766/commits/da0713f8b2fdb0e70227855f62710eedc4d23270)